packages/mobile/e2e/src/usecases/ResetAccount.js

```diff
@@ -1,26 +1,38 @@
-import { enterPinUiIfNecessary, waitForElementId, sleep } from '../utils/utils'
+import { enterPinUiIfNecessary, pixelDiff, waitForElementId } from '../utils/utils'
 import { SAMPLE_BACKUP_KEY } from '../utils/consts'
+import { dismissBanners } from '../utils/banners'
 
 export default ResetAccount = () => {
+  beforeEach(async () => {
+    await device.reloadReactNative()
+    await dismissBanners()
+  })
+
   it('Reset Account by doing the Account Key quiz', async () => {
     // This test has very high flakiness on Android. I did my best to fix it, but
     // locally it works every time but on CI it fails 50%+ of the time.
     // TODO: Keep investigating the source of flakiness of this test on Android.
-    if (device.getPlatform() === 'android') {
-      return
-    }
+    // if (device.getPlatform() === 'android') {
```

Comment:
Yes, I think the reset functionality is not working well and we need to look into it.

Reply:
Agreed we should have an e2e test for this. The current issue is the account shutdown is making it hard for detox to reconnect to the emulator. Hence why we don't actually reset the account and reopen the application.